src/credential/offer/01-start-flow.ts

```diff
@@ -0,0 +1,87 @@
+import * as z from "zod";
+import { Logger, LogLevel } from "../../utils/logging";
+import { stringToJSONSchema } from "../../utils/zod";
+import { InvalidQRCodeError } from "./errors";
+import { CredentialOfferSchema } from "./types";
+
+const CREDENTIAL_OFFER_SCHEMA = "openid-credential-offer://";
+const CREDENTIAL_OFFER_PARAM = "credential_offer";
+const CREDENTIAL_OFFER_URI_PARAM = "credential_offer_uri";
```

Comment:
What do you think about supporting different schemes like [haip](https://openid.net/specs/openid4vc-high-assurance-interoperability-profile-1_0-03.html#section-4.1)?